Wire-iOS/Sources/Components/Buttons/SpinnerButton.swift

```diff
@@ -0,0 +1,83 @@
+
+// Wire
+// Copyright (C) 2020 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+
+/// A button with spinner at the trailing side. Title text is non trancated.
+final class SpinnerButton: Button {
+    ///TODO: get these from design
```

Comment:
A good best practice could be place this UI constant in a separate extension place on top of the class definition, something like this:

```suggestion
    extension SpinnerButton {
    ///TODO: get these from design
    private static let iconSize = StyleKitIcon.Size.tiny.rawValue
    private static let iconInset: CGFloat = 10
    private static let textInset: CGFloat = 5
    private static let inset: CGFloat = 10
}

/// A button with spinner at the trailing side. Title text is non trancated.
final class SpinnerButton: Button { ... }
```

Reply:
I will move them after the design spec is finalized.